Rubberduck.Core/UI/Inspections/InspectionResultsControl.xaml

```diff
@@ -136,6 +136,11 @@
 
                 <Label Content="{Resx ResxName=Rubberduck.Resources.RubberduckUI, Key=GroupingGrid_Filter}" VerticalContentAlignment="Center" />
 
```

Comment:
:bulb: what if there was a dropdown glued to the RHS of the `CopyResultsCommand`, with flatstyle toggles for each supported format? ...and then a setting to automatically select a preferred default.

Not for this PR, but thinking it could be its own XAML control (the button+dropdown combo); the list of buttons could be templated off the enum type itself...